### DISCUSSION TITLE
Roms browsing

### DISCUSSION THREAD

**Discussion by landsat77** [Polprzewodnikowy/N64FlashcartMenu#68] [Ideas]
### Description

Add the ability to browse roms pages using left and right direction

**Comment by Polprzewodnikowy** [comment]
What do you describe as "page" in that context? File browser is just a scrollable list and there isn't a concept of pages. If you meant fast scrolling then you can use up/down C buttons to scroll list by 10 entries.

**Reply by TharathielCB** [reply]
Maybe you could change it to 1 entry by pressing up/down and 10 entries by pressing left/right.

**Comment by landsat77** [comment]
I mean fastest scrolling like **Everdrive** or **ED64+** using left and right. And yes that's mean using pages.

**Comment by networkfusion** [comment]
I kind of think that having pages like Krikzz current menu is a bit cursed. I would much rather use joypad left and right for other needs (like menu about and settings pages). For this menu, folder structures are the way forwards and in future would rather have filter functions for the main selection.

**Comment by networkfusion** [comment]
I have moved this to a discussion as think it is better placed (since the issue did not have enough details.
